dependency/actions/compile/entrypoint.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+
+set -eu
+set -o pipefail
+shopt -s inherit_errexit
+
+readonly PROGDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+function main() {
+  local version output_dir target bundler_dir
+
+  while [ "${#}" != 0 ]; do
+    case "${1}" in
+      --version)
+        version="${2}"
+        shift 2
+        ;;
+
+      --outputDir)
```

Comment:
```suggestion
      --output-dir)
```

Reply:
This flag should match what's outlined in the [README](https://github.com/TisVictress/php-dist/blob/from-frankie2/dependency/actions/compile/README.md).

Reply:
I think this is backwards. The README should have `--outputDir`, that's the flag we want to be using I believe